core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -905,58 +901,27 @@ export class ConversationSkinComponent {
     this.playerPositionService.setDisplayedCardIndex(numCards - 1);
   }
 
-  animateToTwoCards(doneCallback: () => void): void {
-    this.isAnimatingToTwoCards = true;
-    setTimeout(
-      () => {
-        this.isAnimatingToTwoCards = false;
-        if (doneCallback) {
-          doneCallback();
-        }
-      },
-      TIME_NUM_CARDS_CHANGE_MSEC + TIME_FADEIN_MSEC + this.TIME_PADDING_MSEC
-    );
-  }
-
-  animateToOneCard(doneCallback: () => void): void {
-    this.isAnimatingToOneCard = true;
-    setTimeout(() => {
-      this.isAnimatingToOneCard = false;
-      if (doneCallback) {
-        doneCallback();
-      }
-    }, TIME_NUM_CARDS_CHANGE_MSEC);
-  }
-
   isCurrentCardAtEndOfTranscript(): boolean {
     return this.playerTranscriptService.isLastCard(
       this.playerPositionService.getDisplayedCardIndex()
     );
   }
 
   private _addNewCard(newCard): void {
-    this.playerTranscriptService.addNewCard(newCard);
-    const explorationLanguageCode =
-      this.explorationPlayerStateService.getLanguageCode();
-    const selectedLanguageCode =
-      this.contentTranslationLanguageService.getCurrentContentLanguageCode();
-    if (explorationLanguageCode !== selectedLanguageCode) {
-      this.contentTranslationManagerService.displayTranslations(
-        selectedLanguageCode
-      );
-    }
+    this.conversationFlowService.addNewCardAndDisplayTranslations(newCard);
```

Comment:
From the perspective of the skin, this is just addNewCard, right? The service should just return whatever needs to be displayed on the card in the appropriate language?